requirements/base.txt

```diff
@@ -1,3 +1,4 @@
 multimethod >= 1.0, < 2.0
 neo4j >= 4.4.2, < 5.0
 pandas >= 1.0, < 2.0
+pyarrow >= 7.0, < 8.0
```

Comment:
Thanks for the info :) I think the tests are failing since Python 3.6 is not compat with pyarrow 7, so we definitely need to lower it anyway. pyarrow 3 sounds good to me, what do you think @DarthMax ? We'll need to add pyarrow major version to our tox test matrix though

Reply:
Set to >= 3.0 now. Test matrix changes in https://github.com/neo4j/graph-data-science-client/pull/97